app/sanity.client.ts

```diff
@@ -9,8 +9,7 @@ export function getClient(preview?: {token: string}): SanityClient {
     // eslint-disable-next-line no-warning-comments
     // @TODO perspectives require vX for now
     apiVersion: 'X',
-    useCdn: true,
-    token: process.env.SANITY_API_READ_TOKEN,
+    useCdn: true,    
```

Comment:
```suggestion
    useCdn: true,    
     // Since the process.env var doesn't have a NEXT_PUBLIC_ prefix the token will be `undefined` in the browser bundle.
     token: process.env.SANITY_API_READ_TOKEN,
```